test/xcode-specs.js

```diff
@@ -83,4 +83,10 @@ describe('xcode @skip-linux', () => {
     (parseFloat(version)-6.1).should.be.at.least(0);
   });
 
+  it('should get a list of iOS devices', async() => {
+    let devices = await xcode.getConnectedDevices();
+    should.exist(devices);
```

Comment:
Make this a e2e test and try to write a unit test with sinonjs instead (mock support.core.exec). If we get into the habit of doing that that will catch bugs earlier and faster, greatly increasing the team productivity.